build/ci.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash -ev
+
+## Install toolchain
+if [[ "${TRAVIS_DIST}" == "xenial" ]]; then
+    wget -O - https://apt.llvm.org/llvm-snapshot.gpg.key|sudo apt-key add -
+    echo "deb http://apt.llvm.org/xenial/ llvm-toolchain-xenial-${LLVM_VERSION} main" | sudo tee -a /etc/apt/sources.list
+    sudo add-apt-repository --yes ppa:ubuntu-toolchain-r/ppa
+    sudo apt-get update
+    sudo apt-get install linux-headers-"$(uname -r)"
+    sudo apt-get --yes install bison build-essential cmake flex git libclang-common-"${LLVM_VERSION}"-dev libelf-dev libllvm"${LLVM_VERSION}" libz-dev lldb-"${LLVM_VERSION}" llvm-"${LLVM_VERSION}" llvm-"${LLVM_VERSION}"-dev llvm-"${LLVM_VERSION}"-runtime
```

Comment:
Sort of nitpicky, but could you have only this command be in the if statement? You can use ${TRAVIS_DIST} in the line adding it to sources.list.